test/test_cuda.py

```diff
@@ -3702,22 +3702,34 @@ def test_cuda_graph_allocator_propagates_stream(self):
         not TEST_CUDA_GRAPH, "CUDA >= 11.0 or ROCM >= 5.3 required for graphs"
     )
     def test_cuda_graph_tensor_item_not_allowed(self):
-        # Tesnor.item() calls a synchronize which is not allowed in a cudagraph
-        # Valid for CUDA and ROCm
-        def my_func(a: torch.Tensor, b: torch.Tensor, perm: torch.Tensor):
-            idx = perm[0]
-            a[0] *= b[idx]  # should raise an error during capture
-            return a
+        test_script = """\
```

Comment:
Do we not have a cleaner way to run subprocess tests?

Reply:
beats me, this is just pattern matched against all the other subprocess tests in this fiel